scripts/s3_media_upload

```diff
@@ -495,6 +501,17 @@ def main():
         default="STANDARD",
     )
 
+    upload_parser.add_argument(
+        "--sse-customer-key", help="SSE-C key to use",
+    )
+
+    upload_parser.add_argument(
+        "--sse-customer-algo",
+        help="Algorithm for SSE-C, only used if sse-customer-key is also specified",
+        nargs="?",
```

Comment:
I don't think `nargs=?` is what we want here: that lets you specify the option without giving it a value. c.f. https://docs.python.org/3/library/argparse.html#nargs

```
>>> import argparse
>>> p = argparse.ArgumentParser()
>>> p.add_argument("--foo", nargs="?", default="lol")
_StoreAction(option_strings=['--foo'], dest='foo', nargs='?', const=None, default='lol', type=None, choices=None, required=False, help=None, metavar=None)
>>> p.parse_args(["--foo"])
Namespace(foo=None)
>>> p.parse_args([])
Namespace(foo='lol')
>>> p = argparse.ArgumentParser()
>>> p.add_argument("--foo", default="lol")
_StoreAction(option_strings=['--foo'], dest='foo', nargs=None, const=None, default='lol', type=None, choices=None, required=False, help=None, metavar=None)
>>> p.parse_args([])
Namespace(foo='lol')
>>> p.parse_args(["--foo"])
usage: [-h] [--foo FOO]
: error: argument --foo: expected one argument
```

```suggestion
```